src/components/standalone/security/ips/IpsEventList.vue

```diff
@@ -0,0 +1,359 @@
+      <template v-if="events.length > 0">
+        <div class="flex flex-wrap items-center gap-4">
+          <NeTextInput
+            v-model.trim="searchFilter"
+            :placeholder="t('standalone.ips.filter_events')"
+            is-search
+          />
+          <NeDropdownFilter
+            v-model="typeFilter"
+            :clearFilterLabel="t('ne_dropdown_filter.clear_filter')"
+            :label="t('standalone.ips.type')"
+            :openMenuAriaLabel="t('ne_dropdown_filter.open_filter')"
+            :options="typeFilterOptions"
+            kind="radio"
+          />
+          <NeButton kind="tertiary" @click="clearFilters">
+            {{ t('common.clear_filters') }}
+          </NeButton>
+        </div>
+        <NeTable
+          :ariaLabel="t('standalone.ips.event_list_tab')"
+          :sortDescending="sortDescending"
+          :sortKey="sortKey"
+          card-breakpoint="xl"
+        >
+          <NeTableHead>
+            <NeTableHeadCell column-key="timestamp" sortable @sort="onSort">
+              {{ t('standalone.ips.date') }}
+            </NeTableHeadCell>
+            <NeTableHeadCell column-key="action">
+              {{ t('standalone.ips.type') }}
+            </NeTableHeadCell>
+            <NeTableHeadCell column-key="description">
+              {{ t('standalone.ips.description') }}
+            </NeTableHeadCell>
+            <NeTableHeadCell column-key="source_address">
+              {{ t('standalone.ips.source') }}
+            </NeTableHeadCell>
+            <NeTableHeadCell column-key="destination_address">
+              {{ t('standalone.ips.destination') }}
+            </NeTableHeadCell>
+            <NeTableHeadCell column-key="protocol">
+              {{ t('standalone.ips.protocol') }}
+            </NeTableHeadCell>
+            <NeTableHeadCell column-key="sid">
+              {{ t('standalone.ips.table_id_description') }}
+            </NeTableHeadCell>
+            <NeTableHeadCell column-key="actions"></NeTableHeadCell>
+          </NeTableHead>
+          <NeTableBody>
+            <NeTableRow v-if="filteredEvents.length < 1">
+              <NeTableCell colspan="8">
+                <NeEmptyState
+                  :description="t('common.try_changing_search_filters')"
+                  :icon="faCircleInfo"
+                  :title="t('standalone.ips.no_events_found')"
+                  class="bg-white dark:bg-gray-950"
+                >
+                  <NeButton kind="tertiary" @click="clearFilters">
+                    {{ t('common.clear_filters') }}
+                  </NeButton>
+                </NeEmptyState>
+              </NeTableCell>
+            </NeTableRow>
+            <NeTableRow v-for="item in paginatedItems" v-else :key="`${item.id}-${item.timestamp}`">
+              <NeTableCell :data-label="t('standalone.ips.date')" class="text-nowrap">
+                <span :title="item.timestamp">
+                  {{ new Date(item.timestamp).toLocaleString() }}
```

Comment:
This is fine but it uses a different date format from other pages, I'd use `formatDateLoc` with format `PPpp`.